core/array.rbs

```diff
@@ -2182,7 +2182,7 @@ class Array[unchecked out Elem] < Object
   #
   # Related: see [Methods for Querying](rdoc-ref:Array@Methods+for+Querying).
   #
-  def include?: (Elem object) -> bool
+  def include?: (untyped object) -> bool
```

Comment:
Initially, I thought to allow taking nil value. Therefore I changed this type as follows once:

```suggestion
  def include?: (Elem? object) -> bool
              | (untyped object) -> false
```

But I noticed it does not help the numbers' case. Therefore, I used `untyped` here finally.